test/jdk/jdk/incubator/vector/templates/Perf-Scalar-header.template

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2018, 2021, Oracle and/or its affiliates. All rights reserved.
```

We just use one year or the range. Hopefully not too tedious to change with a global search and replace.

Reply:
No worries. Done.